docs/blog/20/5-gettig_started_to_vitis_acceleration_flow_with_zynq_7000.md

```diff
@@ -0,0 +1,351 @@
+---
+date: 2020-12-07
+author: Anıl TIRLIOĞLU
+description: Getting Started to Vitis Acceleration Flow with Zynq 7000.
+axlang: "en"
+---
+
+# Getting Started to Vitis Acceleration Flow with Zynq 7000
+
+We all know, sometimes it is just hard to get start learning things from
+vendor documentations. This tutorial will follow beginner friendly steps to run
+your first accelerator on an FPGA.
+
+!!! info
+    This tutorial will be more beneficial when used in
+    conjunction with the official Xilinx tutorial[^1f] published on github.
+
+There are two different device definitions in Xilinx. One is **embedded
+devices**[^2f],  other one is **data center accelerator cards**[^3f]. Embedded
+device can work standalone. It hosts CPU and FPGA. On the other hand, data
+center accelerator cards needs to be attached to PC through PCIe interface. CPU
+on your host PC will manage these FPGA cards. This tutorial will cover embedded
+device flow with a Zynq 7000 series FPGA board ZC706.
+
+!!! info
+    Note that Xilinx tutorial targets Zynq Ultrascale+ ZCU102 and Alveo U200
+    boards for embedded flow and PCIe attached flow respectively. We will follow
+    embedded flow with ZC706[^4f].
+
+ZC706 board XC7Z045 FFG900 Zynq SoC chip. XC7Z045 is a midrange powerful 7
+series FPGA. Resource capacity is pretty enough to hands on tests with accelerators.
+
+| Resouce            | Quantity             |
+| ------------------ | -----------          |
+| Logic Cells        | 350                  |
+| Block RAM (Mb)     | 19.1                 |
+| DSP Slices         | 900                  |
+
+Some essential terms could be helpful to know before starting to Vitis
+acceleration flow.
+
+| Term               | Description                                                                                                                  |
+| ------------------ | -----------                                                                                                                  |
+| Host               | CPU that runs main function, handles management and other possible sequential tasks, functions etc.                          |
+| Device             | FPGA or board that accelerates functions which are called from host, mostly used in OpenCL definitions for GPU and FPGA      |
+| Kernel             | Region on FPGA that runs functions implemented for FPGA programmable logic, it is also called computing unit                 |
```

Comment:
Kernel yerin adı mı yoksa işin adı mı? Alanı mı gösteriyor yoksa hızlandırıcı parçasını mı? Ben tam bilmiyorum, yorum yazarak kapatabilirsin bu görüşü, değiştir diye demiyorum :rabbit:

Reply:
birbirinden bağımsız düşünmüyorum. region içinde koşan fonksiyon sabit olduğu için.

Reply:
Partial, DFX ile vs karışır mı diye düşünmüştüm de, oki. :+1: